cosim/black-parrot-example/vivado/Makefile

```diff
@@ -8,6 +8,9 @@ WD_FLIST   ?= $(abspath flist.watchdog.vcs)
 TOP_FLIST  ?= $(abspath flist.blackparrot.vcs)
 FLIST      ?= $(WD_FLIST) $(TOP_FLIST)
 
+# Set to MULTICORE for different BP configs
+export SV_DEFINES ?=
```

Comment:
Can we keep any new defines in the bsg_blackparrot_pkg that we generate? It makes it easier to deal with different tool versions / types